api/api/utils/image_proxy/__init__.py

```diff
@@ -64,6 +64,7 @@ def get(
     accept_header: str = "image/*",
     is_full_size: bool = False,
     is_compressed: bool = True,
+    media_provider: str = None,
```

Comment:
Why does this default to None?

Reply:
My intention wasn't to disturb the API we're using here _too much_. Though while working on this I realized we are only calling it in the one place within the media view, so I can make this required!